pkg/changelog/v1/bin/tag-editor

```diff
@@ -16,31 +16,50 @@ TEMP_FILE_CHANGELOG="$(mktemp)"
 # Build the initial CHANGELOG.md section in the editor
 #
 # Add version number to the top of the editor
-echo "v$GIT_HEAD_SEMVER" > "$TEMP_FILE_TAG"
+echo "vX.Y.Z" > "$TEMP_FILE_TAG"
 # Add the previous git tag to the editor
 echo "<!-- Previous tag: $GIT_TAG_LATEST -->" >> "$TEMP_FILE_TAG"
+
 # Add the existing CHANGELOG.md [unreleased] section if file exists
 if [ -f CHANGELOG.md ]; then
+    # If there is the old [unreleased] section, replace it with ## Unreleased header
+    sed -i '' '/^## \[Unreleased\]/,/[unreleased]/d' CHANGELOG.md
+
+    # If there is no Unreleased section, insert the Unreleased section on line 11, and blank line after on line 12
+    if ! grep -q '## Unreleased' CHANGELOG.md; then
+        sed -i '' $'11i\\\n## Unreleased\n' CHANGELOG.md
+        sed -i '' $'12i\\\n\n' CHANGELOG.md
```

Comment:
Could this be done by looking for any "double-hyphen" heading (or similar), rather than hardcoding a line number?

Reply:
Yeah I'll try and get it to look for the first ## header and insert before.